src/DatePicker/DatePicker.js

```diff
@@ -231,7 +231,7 @@ class DatePicker extends Component {
   };
 
   isControlled() {
-    return this.props.hasOwnProperty('value');
+    return this.props.value !== null;
```

Comment:
Thx for the reply, in fact I was using the value prop the wrong way, I now pass a default value to the component instead of a null value and trying to set a default value.
Im going to close this PR and the associated issue.
We might want to correct the code of the timepicker though, as it use a null test (as suggested in this PR).